caffe2/utils/CMakeLists.txt

```diff
@@ -1,15 +1,8 @@
-# TODO: Add ThreadPoolXNNPACK.cc when XNNPACK integration is updated
-# to pass the actual threadpool ptr instead of nullptr.
 if(INTERN_BUILD_MOBILE AND NOT BUILD_CAFFE2_MOBILE)
-  add_definitions(-DUSE_INTERNAL_THREADPOOL_IMPL)
```

Comment:
Now defaulting to the third party version for mobile, hence not USE_INTERNAL_THREADPOOL_IMPL.